templates/agenda/index.html.twig

```diff
@@ -69,20 +69,20 @@
                         <td>{{ agenda.start ? agenda.start|date(format_time, false) : '-' }}</td>
                         <td>{{ agenda.end ? agenda.end|date(format_time, false) : '-' }}</td>
                         <td>{{ agenda.status|trans({}, 'agenda_status') }}</td>
-                        <td class="text-right">
+                        <td class="text-end">
                             <!-- Show button -->
-                            <a href="{{ url('agenda_show', {id: agenda.id}) }}" role="button" class="btn btn-primary" data-toggle="tooltip" data-placement="auto" title="{% trans %}Show agenda{% endtrans %}"><i class="fas fa-eye"></i><span class="sr-only">{% trans %}Show agenda{% endtrans %}</span></a>
+                            <a href="{{ url('agenda_show', {id: agenda.id}) }}" role="button" class="btn btn-primary" data-bs-toggle="tooltip" data-placement="auto" title="{% trans %}Show agenda{% endtrans %}"><i class="fas fa-eye"></i><span class="sr-only">{% trans %}Show agenda{% endtrans %}</span></a>
                             {% if is_granted('ROLE_CASEWORKER') or is_granted('ROLE_ADMINISTRATION') %}
                                 <!-- Delete button -->
                                 {% if agenda.isFinished() %}
-                                    <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete finished agenda{% endtrans %}"><i class="fas fa-trash-alt"></i></button>
+                                    <button class="btn btn-danger" disabled data-bs-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete finished agenda{% endtrans %}"><i class="fas fa-trash-alt"></i></button>
```

Comment:
It seems that tooltips don't work when an element has the attribute `disabled`. I'm not sure if tooltips is the correct method for showing a user why a button is disabled in the first place?

If it is, then https://getbootstrap.com/docs/4.0/components/tooltips/#disabled-elements proposes a fix that seems to work.

Reply:
I agree - Tooltip on a disabled button might not be the best for the user.
Why is the button is only disabled, and not completely hidden. Is there a function in the same context that can enable the button?

Reply:
The button was kept `disabled` with a tooltip, rather than removed, to somehow inform the user of why. At one point i experimented hiding/removing the button and instead adding some text explaining why, but never ended up with a good solution.

In this case (https://github.com/itk-dev/naevnssekretariatet/blob/develop/templates/agenda/index.html.twig#L76-L87) the button is added with a specific tooltip if some condition is met.